### PR TITLE
MM-32227 - Update docs for saved messages

### PR DESCRIPTION
**Pull request by cwarnermm**
Docs ticket: https://mattermost.atlassian.net/browse/MM-32227

Updated:
- Users Guide > Messaging > Flagging Messages
   - Updated all references of "flagging messages" to "saving messages"

Note: Changing the source file will be tracked via https://mattermost.atlassian.net/browse/MM-32672

### CODE REVIEW

**Comment by cwarnermm** [comment]
Note: 1 product documentation page remains as-is for now: https://docs.mattermost.com/deployment/bulk-loading-data-format.html (which matches API product docs: https://api.mattermost.com/#tag/posts/paths/~1users~1{user_id}~1posts~1flagged/get)

**Review by esethna — APPROVED** [review]

File: source/administration/telemetry.rst [hunk 1]
```diff
@@ -124,8 +124,8 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
-  - *Advanced Feature Discovery Diagnostics:* Reacted to a message; favorited or un-favorited a channel; flagged or un-flagged a message; pinned or un-pinned a message; replied to a message; expanded the right-hand sidebar; started or finished a WebRTC video call (only in v5.5 and earlier); created or deleted a personal access token; added or removed post:all or post:channels permission; created a category in the sidebar
+  - *Core Feature Discovery Diagnostics:* Created, edited or deleted a message; posted a message containing a hashtag, link, mention or file attachment; searched for a term; searched for saved posts or recent mentions
+  - *Advanced Feature Discovery Diagnostics:* Reacted to a message; favorited or un-favorited a channel; saved or un-saved a message; pinned or un-pinned a message; replied to a message; expanded the right-hand sidebar; started or finished a WebRTC video call (only in v5.5 and earlier); created or deleted a personal access token; added or removed post:all or post:channels permission; created a category in the sidebar
```
Comment (justinegeffen) [inline source/administration/telemetry.rst:128]:
```suggestion
  - *Advanced Feature Discovery Diagnostics:* Reacted to a message; favorited or unfavorited a channel; saved or unsaved a message; pinned or unpinned a message; replied to a message; expanded the right-hand sidebar; started or finished a WebRTC video call (only in v5.5 and earlier); created or deleted a personal access token; added or removed post:all or post:channels permission; created a category in the sidebar
```

File: source/help/getting-started/creating-teams.rst [hunk 2]
```diff
@@ -59,7 +59,7 @@ Mattermost can be deployed both to a single team and to multiple teams.
 Currently, we recommend deploying to a single team for the following reasons:
 
 * Single team deployments promote communication across the organization. When you add multiple teams, groups can become isolated. 
-* We don’t yet support search or channels across teams, which can impact the cross-team user experience. This includes general searches, flagged posts and recent mentions.
+* We don’t yet support search or channels across teams, which can impact the cross-team user experience. This includes general searches, saved posts and recent mentions.
```
Comment (justinegeffen) [inline source/help/getting-started/creating-teams.rst:62]:
```suggestion
* We don’t yet support search or channels across teams, which can impact the cross-team user experience. This includes general searches, saved posts, and recent mentions.
```

File: source/help/getting-started/messaging-basics.rst [hunk 3]
```diff
@@ -33,10 +33,10 @@ Emoji <https://docs.mattermost.com/help/settings/custom-emoji.html>`__.
 **Attach files** by dragging and dropping them into Mattermost or clicking
 the attachment icon in the text input box.
 
-**Flag messages for follow up** by clicking the flag icon next to the
-message timestamp.
+**Save messages for follow up** by clicking the **Save** icon next to the
```
Comment (justinegeffen) [inline source/help/getting-started/messaging-basics.rst:36]:
```suggestion
**Save messages for follow up** using the **Save** icon next to the
```

File: source/help/messaging/flagging-messages.rst [hunk 4]
```diff
@@ -1,31 +1,28 @@
-Flagging Messages
--------------------
+Saving Messages
+---------------
 
-Flags are a way to mark messages for follow up. Your flags are personal, and cannot be seen by other users. There is no limit to the number of posts you can flag.
+Saving a message helps you mark messages for follow up. Your saved messages are personal, and cannot be seen by other users. There is no limit to the number of posts you can save.
```
Comment (justinegeffen) [inline source/help/messaging/flagging-messages.rst:4]:
```suggestion
Saving a message helps you mark messages for follow up. Your saved messages are personal, and can't be seen by other users. There's no limit to the number of posts you can save.
```

File: source/help/messaging/flagging-messages.rst [hunk 5]
```diff
@@ -1,31 +1,28 @@
...
 
-One way we like to use flags is to mark messages we want to respond to later. It's useful when someone asks for something that requires work. We flag the post, keep doing what we're doing, and then go back to our list of flagged posts. After the work is completed, we can reply back to the post and unflag it to clear it from the list.
+One way we like to use saved messages is to mark messages we want to respond to later. It's useful when someone asks for something that requires work. We save the post, keep doing what we're doing, and then go back to our list of **Saved Posts**. After the work is completed, we can reply back to the post and remove it from the list.
```
Comment (justinegeffen) [inline source/help/messaging/flagging-messages.rst:6]:
```suggestion
One way to use saved messages is to mark messages you want to respond to later. You can save the post in the list of **Saved Posts**, refer back to it when you're able to, and once complete you can remove it from the list.
```

File: source/help/messaging/flagging-messages.rst [hunk 6]
```diff
@@ -1,31 +1,28 @@
...
+To try it out, hover over a message to show the **Save** icon. Click the icon to save the message for follow up.
 
-.. image:: ../../images/flag.png
-  :width: 500px
+.. image:: ../../images/save-message.png
 
-Viewing Flagged Messages
-===========================
+Viewing Saved Messages
+=======================
```
Comment (justinegeffen) [inline source/help/messaging/flagging-messages.rst:16]:
```suggestion
=====================
```

File: source/help/messaging/flagging-messages.rst [hunk 7]
```diff
@@ -1,31 +1,28 @@
...
 
-To view a list of your flagged messages, click the flag beside the search box.
+To view a list of your **Saved Posts**, click the **Save** icon at the top of the channel.
```
Comment (justinegeffen) [inline source/help/messaging/flagging-messages.rst:18]:
```suggestion
To view a list of your **Saved Posts**, select the **Save** icon at the top of the channel.
```

File: source/help/messaging/flagging-messages.rst [hunk 8]
```diff
@@ -1,31 +1,28 @@
...
-  :width: 500px
+.. image:: ../../images/saved-posts.png
 
-Unflagging Messages
-===========================
+Unsaving Messages
+=================
 
-To remove an item from the flagged list, click the flag beside a message to clear it.
+To remove an item from the **Saved Posts** list, click the **Save** icon beside a message to clear it.
```
Comment (justinegeffen) [inline source/help/messaging/flagging-messages.rst:25]:
```suggestion
To remove an item from the **Saved Posts** list, select the **Save** icon next to message to clear it.
```

**Review by justinegeffen — CHANGES_REQUESTED** [review]

**Comment by justinegeffen** [comment]
There's a section that I reworded because it's very different to our usual style of writing - I may have read it out of context though, so happy to bounce that around. 

Otherwise just some minor suggestions. :)

**Comment by cwarnermm** [comment]
@justinegeffen - You're right about that one section reading quite differently. I agree with all of your updates, and thank you for finding additional pre-existing "clicks" that I missed in the first round! :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**